### PR TITLE
Bug/sms format

### PR DESCRIPTION
**Pull request by g-vega-cl**
## Purpose  
- Copy/Pasting text keeps the original formatting
- Add ability to indent text

## Changes and Additions  		 
- In `SMSTitle.tsx` added css `white-space: pre-wrap;`to keep newline formatting
- In `SMSTitle.tsx` added changed `SendInput` `input to `textarea`
- In `SMSTitle.tsx` added css to the  `<div>` parent of `SendInput` and to `showEmoji`so everything stays in place with the `textarea` change

## Open Trello Tickets

-[CSV parsing is broken in some instances; messages received from clients are being parsed into the incorrect columns w/in CSV](https://trello.com/c/vg6quwJu/74-csv-parsing-is-broken-in-some-instances-messages-received-from-clients-are-being-parsed-into-the-incorrect-columns-w-in-csv)

## Tested
- Branch is up to date to Master Branch
- Ran `yarn lint` before MR
- Ran `yarn test` before MR
- Application runs without crashes (`yarn start`)

Proof of fix (MacOS)
Screen Shot 2021-05-26 at 11 25 30

### CODE REVIEW

File: config/env.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-"use strict";
```
Comment (rweald) [inline config/env.js:1]: I'd appreciate in the future if we can not include this type of refactor in existing PRs. Makes it much harder to go through the actual intended change when it also includes somewhat unrelated changes to a ton of files.

**Review by rweald — CHANGES_REQUESTED** [review]
Whoa, this looks like now it maybe contains a ton of different stuff, and also a merge commit from master -> branch. Which commits make the actual changes related to just the SMS formatting bug fix? 

Cesar Vega: Everything before June 1st has to do with the SMS formatting. (before the `Feature/logout button (#19) `commit. 
The Feature/logout button (#19) merge/commit includes the logout button and the inactivity logout.

File: src/components/SMSTile/SMSTile.tsx [hunk 2]
```diff
@@ -145,23 +144,33 @@ interface TextProps {
   type: Texter;
 }
 
+export const formatMessageNewLine = (message: String) => {
+  return message.split("\n").map((i, key) => {
```
Comment (rweald) [inline src/components/SMSTile/SMSTile.tsx:148]: I'm not sure we need to actually include a `div` to make the whitespace work? Would a single div be sufficient with the necessary CSS tweaks to honor newlines? Assuming the string contains the necessary whitespace characters already.

File: config/env.js (hunk 1 above)
Comment (g-vega-cl) [reply config/env.js:1]: Deal, thank you for letting me know.

File: src/components/SMSTile/SMSTile.tsx (hunk 2 above)
Comment (g-vega-cl) [reply src/components/SMSTile/SMSTile.tsx:148]: The string does contain the whitespace characters already. 
Example of how the data works in Mongo:
Screen Shot 2021-06-02 at 12 39 02

Comment (g-vega-cl) [reply src/components/SMSTile/SMSTile.tsx:148]: Also, you were right! I deleted `formatMessageNewLine` and just added `white-space: pre-wrap` to the css.

File: src/components/SMSTile.tsx [hunk 3]
```diff
@@ -252,18 +252,25 @@ const SMSTile: React.FC<SMSProps> = ({
             e.preventDefault();
           }}
         >
-          <div className="field has-addons" style={{ padding: "20px" }}>
-            <div className="control" style={{ width: "100%" }}>
+          <div
+            className="field has-addons"
+            style={{ padding: "20px", height: "75px" }}
```
Comment (rweald) [inline src/components/SMSTile.tsx:257]: This wasn't you, but I think it's hilarious they used a styled component, but then still did inline styles 🤦

**Review by rweald — APPROVED** [review]

**Pull request closed without merging**